modules/nw-ptp-dual-wpc-hardware-config-reference.adoc

```diff
@@ -4,7 +4,7 @@
 
 :_mod-docs-content-type: REFERENCE
 [id="nw-ptp-dual-wpc-hardware-config-reference_{context}"]
```

Comment:
there seems to be a reference to wpc specifically in the document id itself, maybe worth changing to e810

Reply:
Updated as per the suggestion.